libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1146,6 +1160,65 @@ void WaveClip::Resample(int rate, BasicUI::ProgressDialog *progress)
    }
 }
 
+void WaveClip::ApplyStretchRatio()
+{
+   const auto stretchRatio = GetStretchRatio();
+   if (stretchRatio == 1.0)
+      return;
+
+   Finally Do { [this] {
+      this->mClipStretchRatio = 1.0;
+      this->mRawAudioTempo = this->mProjectTempo;
+   } };
+
+   constexpr auto durationToDiscard = 0.0;
+   constexpr auto blockSize = 1024;
```

Comment:
Maybe too small.  Did you see if it helps performance of clip joining to go larger?

Reply:
Ah, this one is confusing. It's not related to analysis/synthesis size. The result will be bit-exact independently on it. It's just to limit the amount of memory in our buffer. Would `bufferSize` be better?